app/controllers/admin/base_controller.rb

```diff
@@ -49,8 +49,8 @@ def preview_design_system?(next_release: false)
   end
   helper_method :preview_design_system?
 
-  def render_design_system(design_system_view, legacy_view, next_release: false)
-    if preview_design_system?(next_release:)
+  def render_design_system(design_system_view, legacy_view, layout: "admin")
+    if preview_design_system?(next_release: layout == "design_system")
```

Comment:
I think we might be actually able to simplify this to just:

```suggestion
  def render_design_system(design_system_view, legacy_view)
    if self.respond_to?(:get_layout) && get_layout == "design_system"
```

Rather than passing around the `next_release` or `layout` values at all – because I think that's already worked out by the `get_layout` method of each controller, right?

For example, the specific actions and user permissions are already tested here:

https://github.com/alphagov/whitehall/blob/b20bc5d3c1927e34695ffc681c64987de1ed414a/app/controllers/admin/corporate_information_pages_controller.rb#L30-L39

Reply:
yea great shout. Have implemented it 👍

Reply:
Just realised this actually needs to be as it returns false 

```
self.class.private_method_defined?(:get_layout) && get_layout == "design_system"
```